src/gui/guiFormSpecMenu.cpp

```diff
@@ -133,11 +135,11 @@ GUIFormSpecMenu::~GUIFormSpecMenu()
 }
 
 void GUIFormSpecMenu::create(GUIFormSpecMenu *&cur_formspec, Client *client,
-	JoystickController *joystick, IFormSource *fs_src, TextDest *txt_dest)
+	JoystickController *joystick, IFormSource *fs_src, TextDest *txt_dest, std::string formspecPrepend)
```

Comment:
Line too long and `const std::string &formspec_prepend` (reference + variable naming fix)